pkg/cmd/operator/webhooks_test.go

```diff
@@ -0,0 +1,364 @@
+// Copyright (c) 2021 ScyllaDB
+
+package operator
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"math/big"
+	"net"
+	"net/http"
+	"os"
+	"reflect"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/scylladb/scylla-operator/pkg/genericclioptions"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apiserver/pkg/server/dynamiccertificates"
+)
+
+func TestComplete(t *testing.T) {
+	now := time.Now()
+	_, cert, privateKey, _, _, err := createCertificates(now)
+	if err != nil {
+		panic(err)
```

Comment:
doesn't it sort of imply that we're expecting errors, while we don't really want to test the behaviour of createCertificates?

Reply:
I don't think it does. I don't want our test binary to panic just because it can't create a file. Disk can be full, the test should fail, the others that don't need disk should continue on.